standard-contracts/package.json

```diff
@@ -29,7 +29,7 @@
     "@latticexyz/store": "2.2.8",
     "@latticexyz/world": "2.2.8",
     "@eveworld/world": "0.0.17",
-    "@eveworld/world-core": "0.0.2",
```

Comment:
is it possible not to link?

Reply:
This package is not published in npm since its a internal package. 
if this is published yes we could use the released one

Reply:
nx should resolve the local one

Reply:
it was weird how the ci/cd worked in the previous build without the package being published